appveyor.yml

```diff
@@ -27,43 +43,81 @@ install:
-      -DPLUGIN_ARCHIVE=NO
-      -DPLUGIN_BLACKHOLE=NO
-      -DPLUGIN_CASSANDRA=NO
-      -DPLUGIN_CONNECT=NO
-      -DPLUGIN_CSV=NO
-      -DPLUGIN_EXAMPLE=NO
-      -DPLUGIN_FEDERATED=NO
-      -DPLUGIN_FEDERATEDX=NO
-      -DPLUGIN_HEAP=NO
-      -DPLUGIN_INNOBASE=YES
-      -DPLUGIN_MYISAM=NO
-      -DPLUGIN_MYISAMMRG=NO
-      -DPLUGIN_OQGRAPH=NO
-      -DPLUGIN_PERFSCHEMA=NO
-      -DPLUGIN_SEQUENCE=NO
-      -DPLUGIN_SPHINX=NO
-      -DPLUGIN_SPIDER=NO
-      -DPLUGIN_TEST_SQL_DISCOVERY=NO
-      -DPLUGIN_TOKUDB=NO
-      -DPLUGIN_XTRADB=NO
-      -DWITH_UNIT_TESTS=OFF
-      -DWITH_MARIABACKUP=OFF
-      -DGRN_WITH_BUNDLED_MECAB=ON
-  - cmake --build . --config Debug
-
 test: off
+
+for:
+  -
+    matrix:
+      only:
+        - configuration: BuildCheck
+    build_script:
+      - "echo # > win\\packaging\\CMakeLists.txt"
+      - cmake . -G "%CMAKE_GENERATOR_NAME%"
+          -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          -DPLUGIN_ARCHIVE=NO
+          -DPLUGIN_BLACKHOLE=NO
+          -DPLUGIN_CASSANDRA=NO
+          -DPLUGIN_CONNECT=NO
+          -DPLUGIN_CSV=NO
+          -DPLUGIN_EXAMPLE=NO
+          -DPLUGIN_FEDERATED=NO
+          -DPLUGIN_FEDERATEDX=NO
+          -DPLUGIN_HEAP=NO
+          -DPLUGIN_INNOBASE=YES
+          -DPLUGIN_MYISAM=NO
+          -DPLUGIN_MYISAMMRG=NO
+          -DPLUGIN_OQGRAPH=NO
+          -DPLUGIN_PERFSCHEMA=NO
+          -DPLUGIN_SEQUENCE=NO
+          -DPLUGIN_SPHINX=NO
+          -DPLUGIN_SPIDER=NO
+          -DPLUGIN_TEST_SQL_DISCOVERY=NO
+          -DPLUGIN_TOKUDB=NO
+          -DPLUGIN_XTRADB=NO
+          -DWITH_UNIT_TESTS=OFF
+          -DWITH_MARIABACKUP=OFF
+          -DGRN_WITH_BUNDLED_MECAB=ON
+      - cmake --build . --config RelWithDebInfo
+  -
+    matrix:
+      only:
+        - configuration: ReleasePackage
+    build_script:
+      - set
+      - cd %APPVEYOR_BUILD_FOLDER%\..
+      - git clone --depth 1
+        https://github.com/cosmo0920/PowerShell-for-Mroonga-building.git mrnbuild
+      - cd mrnbuild/powershell
+      - set "PATH=C:\msys64\usr\bin;C:\msys64\mingw64\bin;C:\Windows\System32;C:\Windows;%PATH%"
+      - set MSYSTEM=MINGW64
+      - pacman -Sy --noconfirm bison
+      - ps: .\get-mroonga-batfiles.ps1
+      - ps: .\prepare-building-mroonga.ps1
+      - cmake --version
+      - bison --version
+      - move ..\..\mariadb-%MARIADB_VERSION% work/source
+      - cd .\work
+      - if "%MROONGA_PLATFORM%" == "win32" call .\build-vc2015-zip-32.bat
+      - if "%MROONGA_PLATFORM%" == "winx64" call .\build-vc2015-zip-64.bat
+      - 7z x mariadb-%MARIADB_VERSION%-%MROONGA_PLATFORM%.zip
+      - cd ..\
+      - netsh advfirewall firewall add rule name="mariadb" dir=in action=allow protocol=TCP localport=3306 enable=yes profile=any
+      - ps: .\install-mroonga.ps1 -mariadbVersion $env:MARIADB_VERSION -platforms $env:MROONGA_PLATFORM
+      - ps: .\create-mrnzip.ps1 -mariadbVersion $env:MARIADB_VERSION -mroongaVersion $env:MROONGA_VERSION -platforms $env:MROONGA_PLATFORM
```

Comment:
Or we can use 7z like we did in Groonga.

Reply:
Should we move archiving command to `on_success`?

Reply:
I've moved it.